### DISCUSSION TITLE
Timer wait between task transiction

### DISCUSSION THREAD

**Discussion by carlsonwf** [NitorCreations/nflow#664] [General]
I am still not using nFlow, just studying it.

I have a concrete workflow that, after starting, I have to wait 180 days to start the next task. How can I model this timer wait with nFlow?

**Comment by RealZimboGuy** [comment]
very easy 

		return NextAction.moveToStateAfter(State.myState,
				DateTime.now().plusDays(180),
				"wait for 180 days then exexute this state");
